Content/Onboarding/onboarding.json

```diff
@@ -13,5 +13,15 @@
 
 	"onboarding_eula_headerTitle": "User Terms",
 	"onboarding_eula_headerSubtitle": "To proceed, you must accept the user terms below.",
-	"onboarding_eula_accept": "Accept"
+	"onboarding_eula_accept": "Accept",
+
+	"onboarding_cloudAndroid_backupTitle": "Back up your wallet settings",
+	"onboarding_cloudAndroid_backupSubtitle": "Connect to Google Drive to automatically backup your Radix wallet settings.",
+	"onboarding_cloudAndroid_skip": "Skip",
+	"onboarding_cloudAndroid_backupButton": "Back up to Google Drive",
+
+	"onboarding_cloudAndroid_restore_backupTitle": "Restore wallet from backup",
+	"onboarding_cloudAndroid_restore_backupSubtitle": "Log in to Google Drive to restore your Radix Wallet from backup.",
+	"onboarding_cloudAndroid_restore_skip": "Skip",
+	"onboarding_cloudAndroid_restore_loginButton": "Log in to Google Drive"
```

Comment:
These still have 4 levels

Reply:
oh right, i missed them in front of my eyes 🤦🏽